src/svg2pdf.js

```diff
@@ -1888,8 +1888,32 @@ SOFTWARE.
     var alignmentBaseline = getAttribute(node, "vertical-align") || getAttribute(node, "alignment-baseline");
     if (alignmentBaseline) {
       var matchArr = alignmentBaseline.match(/(baseline|text-bottom|alphabetic|ideographic|middle|central|mathematical|text-top|bottom|center|top|hanging)/);
-      if (matchArr) {
-        attributeState.alignmentBaseline = matchArr[0];
+      switch (matchArr) {
```

Comment:
Don't save the "mapped" property in the `attributeState` (it is meant to mirror the SVG property state). Instead, use the mapped properties only for the `pdf.text(...)` call. Instead of the switch you could also use an object:
```
{
  "text-bottom": "bottom",
  ...
}
```